binding/icon.go

```diff
@@ -0,0 +1,14 @@
+package binding
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/widget"
+)
+
+func BindIconResource(icon *widget.Icon, data *ResourceBinding) {
```

Comment:
Here there is only one type, so even more than above I wonder if it could be BindIcon(...)